storio-sqlite/src/test/java/com/pushtorefresh/storio/sqlite/integration/InterceptorTest.java

```diff
@@ -0,0 +1,314 @@
+                        .table(TweetTableMeta.TABLE)
+                        .build()
+                )
+                .prepare()
+                .executeAsBlocking();
+        checkInterceptorsCalls();
+    }
+
+    @Test
+    public void getNumberOfResultsWithRawQuery() {
+        storIOSQLite.get()
+                .numberOfResults()
+                .withQuery(RawQuery.builder()
+                        .query("select * from " + TweetTableMeta.TABLE)
+                        .build()
+                )
+                .prepare()
+                .executeAsBlocking();
+        checkInterceptorsCalls();
+    }
+
+    @Test
+    public void getNumberOfResultsWithQuery() {
+        storIOSQLite.get()
+                .numberOfResults()
+                .withQuery(Query.builder()
+                        .table(TweetTableMeta.TABLE)
+                        .build()
+                )
+                .prepare()
+                .executeAsBlocking();
+        checkInterceptorsCalls();
+    }
+
+    @Test
+    public void getObjectWithRawQuery() {
+        storIOSQLite.get()
+                .object(Tweet.class)
+                .withQuery(RawQuery.builder()
+                        .query("select * from " + TweetTableMeta.TABLE)
+                        .build()
+                )
+                .prepare()
+                .executeAsBlocking();
+        checkInterceptorsCalls();
+    }
+
+    @Test
+    public void getObjectWithQuery() {
+        storIOSQLite.get()
+                .object(Tweet.class)
+                .withQuery(Query.builder()
+                        .table(TweetTableMeta.TABLE)
+                        .build()
+                )
+                .prepare()
+                .executeAsBlocking();
+        checkInterceptorsCalls();
+    }
+
+    @Test
+    public void putCollection() {
+        storIOSQLite.put()
+                .objects(Collections.singleton(createTweet()))
+                .prepare()
+                .executeAsBlocking();
+        checkInterceptorsCalls();
+    }
+
+    @Test
+    public void putContentValues() {
+        storIOSQLite.put()
+                .contentValues(createContentValues())
+                .withPutResolver(createCVPutResolver())
+                .prepare()
+                .executeAsBlocking();
+        checkInterceptorsCalls();
+    }
+
+    @Test
+    public void putContentValuesIterable() {
+        storIOSQLite.put()
+                .contentValues(createContentValues(), createContentValues())
+                .withPutResolver(createCVPutResolver())
+                .prepare()
+                .executeAsBlocking();
+        checkInterceptorsCalls();
+    }
+
+    @Test
+    public void putObject() {
+        storIOSQLite.put()
+                .object(createTweet())
+                .prepare()
+                .executeAsBlocking();
+        checkInterceptorsCalls();
+    }
+
+    private void checkInterceptorsCalls() {
+        assertThat(callCount.get()).isEqualTo(2);
```

Comment:
Sorry, I should update robolectric to support mockito2 (https://github.com/robolectric/robolectric/issues/2677). I will done it soon.